packages/mg-hubspot-api/lib/processor.js

```diff
@@ -246,17 +247,21 @@ module.exports.processContent = (html, postUrl, errors) => {
  * }
  */
 module.exports.processPost = (hsPost, tags, errors) => {
+    // Get the current timestamp - https://date-fns.org/docs/getTime
+    const timestampNow = getTime(new Date());
```

Comment:
Is there a reason why we're using `getTime` here and not `formatISO` like we do with the other packages?

Reply:
HubSpot returns timestamps in their API so I followed that pattern. But you're absolutely right, this should use `formatISO` for consistency.

Reply:
At least I thought that was true. I can't see if we're using v2 or v3 of their API, but given no version is specified I'll assume it's v3. One uses timestamps, the other ISO 8601 dates‽ Either way, will update.

Reply:
Updated!